tests/test-suite.test.ts

```diff
@@ -80,6 +81,25 @@ describe('Passage API Requests', () => {
             const user = await passage.user.get(userID);
             expect(user).toHaveProperty('id', userID);
         });
+        test('getUserByIdentifier', async () => {
+            const randomEmail = faker.internet.email();
+            const createdUserWithEmail = await passage.user.create({
+                email: randomEmail,
+            } as CreateUserRequest);
+            expect(createdUserWithEmail).toHaveProperty('email', randomEmail.toLowerCase());
+
+            const userByIdentifier = await passage.user.getUserByIdentifier(randomEmail);
```

Comment:
An additional test case for phone number would be good to have

Reply:
added this test